quickstarts/WFLY-16140_Remove_outdated_Quickstarts.adoc

```diff
@@ -0,0 +1,231 @@
+*** Removal justification: This quickstart is just a client to use with the helloworld-rs quickstart, and we believe that the jaxrs-client quickstart is better as a basic REST quickstart, and it's enough to just keep that one.
+
+** spring-greeter
+*** Description: The `spring-greeter` quickstart is based on the `greeter` quickstart, but differs in that it uses Spring MVC for Mapping `GET` and `POST` requests.
+*** Removal justification: We just need one Spring related quickstart, and the spring-resteasy one is the best one to keep.
+
+** spring-kitchensink-basic
+*** Description: The `spring-kitchensink-basic` quickstart is an example of a Jakarta EE application using JSP, JPA and Spring 5.x.
+*** Removal justification: We just need one Spring related quickstart, and the spring-resteasy one is the best one to keep.
+
+** spring-kitchensink-springmvctest
+*** Description: The  `spring-kitchensink-springmvctest` quickstart demonstrates how to create an MVC application using JSP, JPA and Spring 5.x.
+*** Removal justification: We just need one Spring related quickstart, and the spring-resteasy one is the best one to keep.
+
+** tasks-rs
+*** Description: The `tasks-rs` quickstart demonstrates how to implement a JAX-RS service that uses JPA persistence.
+*** Removal justification: This quickstart is a JAXRS variant of the tasks-jsf quickstart, which (unlike this one) is OpenShift compatible and we will keep.
+
+** websocket-client
+*** Description: Demonstrates use of a Javascript WebSocket client, WebSocket configuration, programmatic binding, and secure WebSocket.
+*** Removal justification: This quickstart is redundant considering the websocket functionality covered, we will keep just the websocket-hello and websocket-endpoint quickstarts.
+
+** xml-jaxp
+*** Description: The `xml-jaxp` quickstart demonstrates how to use Servlet and JSF to upload an XML file to WildFly/JBoss EAP and validate and parse it using DOM or SAX.
+*** Removal justification: The topics covered by this quickstart are not really related with WildFly/EAP, and not worth to cover in developer guide (file upload, xml parsing).
+
+=== Hard Requirements
+
+* N/A
+
+=== Nice-to-Have Requirements
+
+* N/A
+
+=== Non-Requirements
+
+* N/A
+
+== Implementation Plan
+
+*
+
+== Test Plan
+
+*
+
+== Community Documentation
+
+* The Quickstart's community documentation is automatically built, and the removal of quickstarts requires no manual update.
+
+== Release Note Content
+
+The following Quickstarts were removed, due to being outdated and/or redundant:
+** app-client
+** bean-validation
+** ejb-asynchronous
+** ejb-in-ear
+** ejb-in-war
+** ejb-security
+** ejb-security-jaas
+** greeter
+** helloworld-html5
+** helloworld-mbean
+** helloworld-rs
+** helloworld-ssl
+** inter-app
+** jaxws-addressing
+** jaxws-pojo
+** jts-distributed-crash-rec
+** kitchensink-angularjs
+** kitchensink-ear
+** kitchensink-jsp
+** kitchensink-ml
+** logging-tools
+** managed-executor-service
```

Comment:
The following QS are missing: 
messaging-clustering
payment-cdi

Reply:
I'm pretty sure I had those, maybe got lost in an evil rebase... Anyway re-added, thanks